hokusai/services/docker.py

```diff
@@ -1,13 +1,15 @@
 import os
 
+from hokusai import CWD
 from hokusai.lib.config import config
 from hokusai.lib.common import shout
 from hokusai.lib.exceptions import HokusaiError
 
 class Docker(object):
   def build(self):
-    docker_compose_yml = os.path.join(os.getcwd(), 'hokusai/build.yml')
-    legacy_docker_compose_yml = os.path.join(os.getcwd(), 'hokusai/common.yml')
+    docker_compose_yml = os.path.join(CWD, 'hokusai/build.yml')
```

Comment:
should these switch to `HOKUSAI_CONFIG_DIR`?

Reply:
DRY FTW